src/probes/crash_probe.c

```diff
@@ -443,12 +444,18 @@ static bool is_banned_path(char *fullpath)
         }
 
         // Anything outside of /usr/, or in /usr/local/, we consider third-party
-        if (g_regex_match_simple("^[!/]usr[!/]", fullpath, 0, 0) == FALSE) {
-                return true;
-        } else if (g_regex_match_simple("^[!/]usr[!/]local[!/]", fullpath,
-                                        0, 0) == TRUE) {
-                return true;
-        }
+
+	if (strncmp(fullpath, "/usr/", 5) != 0)
```

Comment:
what about create a struct with all test and its expected result

```
struct test {
    const char path[PATH_MAX];
    bool expected;
} tests [] = {
   {  .path = "!usr!", .expected = true },
   {  .path = "/usr/local/", .expected = false },
};
```
and then use a ```for``` to iterate and run all tests